Wrappers/Python/ccpi/viewer/CILViewer2D.py

```diff
@@ -329,13 +329,17 @@ def ChangeOrientation(self, new_slice_orientation):
         self.UpdatePipeline(True)
 
     def OnKeyPress(self, interactor, event):
+        al = self._viewer.axisLabelsText
```

Comment:
why is the member `axisLabelsText` required?

Reply:
answer below